### PR TITLE
Enable driver for Realtek card readers

### LINKED ISSUE #2688: Intel NUC SD Card Reader doesn't work
### Describe the issue you are experiencing

I wanted to move backups to an SD Card instead of the NUC SSD but I've found the module for the SDCard reader is not present on the OS.

### What operating system image do you use?

generic-x86-64 (Generic UEFI capable x86-64 systems)

### What version of Home Assistant Operating System is installed?

10.4

### Did you upgrade the Operating System.

Yes

### Steps to reproduce the issue

1. Insert an SDCARD
2. See that nothing happens
3. Check dmesg and see that is not being detected.

### Anything in the Supervisor logs that might be useful for us?

```txt
N/A
```

### Anything in the Host logs that might be useful for us?

```txt
N/A
```

### System information

_No response_

### Additional information

```bash
# lspci | grep 10ec:5229
01:00.0 Class ff00: 10ec:5229
# lsmod | grep rtsx_pci

# modprobe rtsx_pci
modprobe: FATAL: Module rtsx_pci not found in directory /lib/modules/6.1.39
```

Kernel module: https://cateee.net/lkddb/web-lkddb/MISC_RTSX_PCI.html

### PR DESCRIPTION
**Pull request by sairon**
Enable PCI card reader found on some Intel NUC models, along with the USB drivers as well.

Adds two new modules (listed with size):

30104   /lib/modules/6.1.68-haos/kernel/drivers/misc/cardreader/rtsx_usb.ko
167240  /lib/modules/6.1.68-haos/kernel/drivers/misc/cardreader/rtsx_pci.ko

Fixes #2688

### CODE REVIEW

File: buildroot
Comment (agners) [inline buildroot:1]: This change is not related to this commit no?

Comment (sairon) [reply buildroot:1]: Yeah, not really, I just had some mess in my branches obviously. Fixed.

**Review by agners — APPROVED** [review]
